inspirehep/modules/workflows/tasks/actions.py

```diff
@@ -239,10 +238,9 @@ def stop_in_error_if_record_not_valid(obj, eng):
     try:
         validate(obj.data, 'hep')
     except ValidationError as err:
-        obj.log.error(err.message)
-        obj.extra_data['_error_msg'] = err.message
-        obj.status = ObjectStatus.ERROR
-        raise WorkflowsError('The record contained in the workflow is not schema compliant.')
+        message = 'The record contained in the workflow is not schema compliant: \n' + err.message
+        error_workflow_fct = error_workflow(message)
+        error_workflow_fct(obj, eng)
```

Comment:
Calling a workflow step in a workflow step is possible, but probably not a good idea...